nix/cudaPackagesExtensions/0001-add-latest-nccl.nix

```diff
@@ -0,0 +1,13 @@
+{ fetchFromGitHub }:
+ffinal: pprev: {
+  nccl_latest = pprev.nccl.overrideAttrs (prevAttrs: {
```

Comment:
It doesn't look like `prevAttrs` is being used -- remember that `overrideAttrs` takes three different types of argument:

- attribute set (unconditional overrides)
- function to attribute set (uses previous values)
- function to function to attribute set (uses previous and final values)